client/main.lua

```diff
@@ -22,50 +21,49 @@ end)
 CreateThread(function()
     if Config.UseTarget then
         for key, value in pairs(Config.PawnLocation) do
-            exports['qb-target']:AddBoxZone('PawnShop'..key, value.coords, value.length, value.width, {
-                name = 'PawnShop'..key,
-                heading = value.heading,
-                minZ = value.minZ,
-                maxZ = value.maxZ,
-                debugPoly = value.debugPoly,
-            }, {
+            exports.ox_target:addBoxZone({
+                coords = value.coords,
+                size = value.size,
+                rotation = value.heading,
+                debug = value.debugPoly,
                 options = {
                     {
-                        type = 'client',
+                        name = 'PawnShop' .. key,
                         event = 'qb-pawnshop:client:openMenu',
                         icon = 'fas fa-ring',
-                        label = 'Pawn Shop',
-                    },
-                },
-                distance = value.distance
+                        label = 'PawnShop' .. key,
```

Comment:
```suggestion
                        label = 'PawnShop ' .. key,
```
Would be better looking with a space there